### ISSUE TITLE
[Cesium3DTileStyleEngine] Missing support of access to values which are not in batch table, but present

### ISSUE THREAD

**Issue by SunBlack** [AnalyticalGraphicsInc/cesium#4654]
It is currently not possible to access coordinates from a Cesium3DTileStyle.

An example would be:
```
tileset.style = new Cesium3DTileStyle({
	"color" : {
		"conditions" : [
			["(${x} < ${offset} || ${x} > ${offset} + ${maxHeight})", "rgb(255, 255, 255)"],
			[true, "rgb((${x} - ${offset})/${maxHeight}, 230, 230)"],
		]
	}
});
```

It would be also nice if you have access to BoundingBoxes and other properties of a tileset, so a support of following attributes would be nice:
* x/y/z
* bounding box of current tile
* bounding box of root tileset
* level in tree of current tile, starting from root tileset

**Comment by lilleyse** [comment]
Thanks for the suggestions, I will add them to the styling roadmap.

For point clouds, accessing the position is already available. The spec hasn't been updated yet, but here is a draft:

> `${POSITION}` is an array of three values representing the xyz coordinates of the point before the `RTC_CENTER` and tile transform are applied. When the positions are quantized, `${POSITION}` refers to the position after the `QUANTIZED_VOLUME_SCALE` is applied, but before `QUANTIZED_VOLUME_OFFSET` is applied.

There are also a few examples in the Sandcastle demo: 
http://cesium-dev.s3-website-us-east-1.amazonaws.com/cesium/3d-tiles/Apps/Sandcastle/index.html?src=3D%20Tiles%20Point%20Cloud%20Styling.html&label=Showcases

**Comment by SunBlack** [comment]
Well thanks for the hint with ${POSITION}. But without access to any offset it is currently not really usefull with point clouds (or I need to adjust each tile to have same RTC-Center ;-) ).

**Comment by lilleyse** [comment]
Yeah pretty much, we've wanted to support bounding volume information in the styling for a while.

**Comment by pjcozzi** [comment]
>  we've wanted to support bounding volume information in the styling

CC https://github.com/AnalyticalGraphicsInc/3d-tiles/issues/127
